swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -9,12 +9,15 @@
 import Foundation
 import FullStory
 
-/**
+/*
  FullStory Manager is a file full of functions and enums that can make the FullStory API easier to apply to areas of your project.
+    - Identifying a user
     - create events
-    - logging and unmasking elements
+    - logging
+    - masking, unmasking, and excluding elements
  */
 
+/// We created an enum that represents the major events in the iOS app. (This could be different for you based on the flow of your app.)
```

Comment:
Lead with the purpose like, "The `Event` enum represents major events in an iOS app. These may differ based on your particular app."  On the last point, it's worth consider how opinionated FullStory Manager will be.  If my custom app has an event not in this enumeration, do I have to edit the code, does it matter, can I register it from the outside?